### DISCUSSION TITLE
Android12兼容移除动画

### DISCUSSION THREAD

**Discussion by bada0520** [Tencent/libpag#1884] [Q&A]
之前项目使用的是pag 4.2版本， pagView提供了重写getAnimationScale的能力，所以即使开启了移除动画选项时，依然可以播放pag动画，但在pag4.3.33版本，不再使用ValueAnimator实现，而是用PAGAnimator实现，当开启移除动画选项时无法播放了，请问有没有办法规避？

**Comment by kevingpqi123** [comment]
PAG 4.2 中 getAnimationScale 只是为了获取动画伸缩参数，如果参数是0，直接触发 onAnimationEnd 方法回调，你上面指的是啥，没太理解

**Comment by CodeJhF** [comment]
这条我们在当时的回复里已经给出结论，这里就不重复贴了，先关闭本条。
如果按当时的结论排查后仍有问题，欢迎回复补充最小复现 demo 与版本号（4.4 / 4.5），我们会重新看。（stale-close）
